pom.xml

```diff
@@ -54,7 +54,7 @@ under the License.
 
     <properties>
         <aws.sdkv1.version>1.12.439</aws.sdkv1.version>
-        <aws.sdkv2.version>2.20.144</aws.sdkv2.version>
+        <aws.sdkv2.version>2.26.19</aws.sdkv2.version>
```

Comment:
Do we need to update any `NOTICE` files?

Reply:
Yes, we need to also update all mentions of the SDK versions in `NOTICE` files

Reply:
Thank you, updated